qutebrowser/mainwindow/tabwidget.py

```diff
@@ -541,6 +537,9 @@ def tabSizeHint(self, index):
                     index < no_pinned_width % no_pinned_count):
                 width += 1
 
+            # If we have a width too small, force it to be at least the minimum_size
```

Comment:
@jgkamat Could you split the comment in multiple lines?
C:540, 0: Line too long (84/79) (line-too-long)